vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/network/apis/restrictedendpoints/register.go

```diff
@@ -7,6 +7,16 @@ import (
 
 var GroupVersion = schema.GroupVersion{Group: "network.openshift.io", Version: runtime.APIVersionInternal}
 
+// Kind takes an unqualified kind and returns back a Group qualified GroupKind
+func Kind(kind string) schema.GroupKind {
```

Comment:
where is this called?

Reply:
I am unsure if `Kind()` and `Resource()` methods are being used. I was following precedence of other admission plugins. But `GetObjectKind()` seems to be necessary for type registration.